static/js/checks.js

```diff
@@ -305,4 +311,75 @@ $(function() {
         var text = e.trigger.getAttribute("data-clipboard-text");
         prompt("Press Ctrl+C to select:", text);
     });
-});
\ No newline at end of file
+
+    // Shows set priority modal
+    $(".check-priority").click(function(e) {
+        var $this = $(this);
+        var a = e.target;
+        var priority = a.getAttribute("data-priority");
+        var emails = a.getAttribute("data-escal-emails");
+        console.log(priority, emails)
```

Comment:
`console.log()` should only be used for debugging purposes. For production ready code, it should be eliminated. The same comment applies to all your `.js` files.

Reply:
Done. Removed console.log()